database_files/hvac_minimum_requirement_water_heaters_90_1.json

```diff
@@ -1,12 +1,12 @@
 [
     {
         "template": "90.1-2004",
```

Comment:
@lzwang26  - Could you please revert the changes in this file? They are going to create conflicts when #2 is merged.